cupy/cuda/__init__.py

```diff
@@ -51,11 +51,13 @@
 except ImportError:
     nccl_enabled = False
 
-try:
-    from cupy.cuda import cutensor  # NOQA
-    cutensor_enabled = True
-except ImportError:
-    cutensor_enabled = False
+cutensor_enabled = False
+if int(os.getenv('CUTENSOR_DISABLED', 0)) == 0:
+    try:
+        from cupy.cuda import cutensor  # NOQA
+        cutensor_enabled = True
+    except ImportError:
+        pass
```

Comment:
Could you exclude the change to `cupy/cuda/__init__.py` unless it is necessary?

Reply:
OK, I will remove the change.